docs/contributing/contribution/merge.md

```diff
@@ -0,0 +1,102 @@
+# Integration Branch (Milestone) Merge Guide
+
+```mermaid
+flowchart TB
+  fix((Create fix Issue))
+  pr((Create a PR<br>to main))
+  pr --> testing{Automated<br>Regression}
+  testing --> |error| fix
+  testing --> |success| qa[Manual QA<br>Enforcement]
+  qa --> |comment| review{Comments<br>Resolved?}
+  
+  review --> |No| fix
+	feedback --> |answer| review
+
+  review --> |Yes| approvals{Actors<br>Approved?}
+  approvals --> |No| fix
+  approvals --> |Yes| merge[Merge PR]
+  merge --> close((Close Milestone<br>& Issues))
+```
+
+Once all the issues are finalized, and the milestone integration branch is ready and tested, a PR has to be open against the `main` branch. You can follow this guide to create the PR,
+
+- **From a fork:** [https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork)
+- **From a branch:** [https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)
+
+On the PR, please include any details related to the job that you pushed. Proof of your local tests passing is a good practice, to help other developers know that the code is tested. Any screenshot or screencast is welcome to help others understand the impact of your work.
+
+Be as detailed as you can, since this will help get a faster review of the PR. If the PR is not being reviewed, make sure to tag the issue creator from time to time, and even propose a review meeting to make this faster.
+
+If the issue had extra deliverables, make sure to include them as part of your PR, to get the approvals.
+
+Every PR should follow this template,
+
+```jsx
```

Comment:
1. `s/jsx/md`?
2. Ditto my comment about linking to the template (or even embedding it if Github supports it?) so we won't have to maintain two sources of truth?

Reply:
Won't do, changing template MD for links